components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -176,42 +176,40 @@ function renderIdeOption(
     version: IDEOption["imageVersion"],
     onSelect: () => void,
 ): JSX.Element {
-    const label = option.type === "desktop" ? "" : option.type;
+    const shouldShowOptionType = option.type !== "desktop" || option.title === "VS Code"; // Force show of "Desktop" in the list for VS Code Desktop
     const card = (
         <SelectableCardSolid className="w-36 h-44" title={option.title} selected={selected} onClick={onSelect}>
-            <div className="flex justify-center mt-3">
-                <img className="w-16 filter-grayscale self-center" src={option.logo} alt="logo" />
-            </div>
-            <div
-                className="mt-2 px-3 py-1 self-center"
-                style={{
-                    minHeight: "1.75rem",
-                }}
-            >
-                {label ? (
-                    <span
-                        className={`font-semibold text-sm ${
-                            selected ? "text-gray-100 dark:text-gray-600" : "text-gray-600 dark:text-gray-500"
-                        } uppercase`}
-                    >
-                        {label}
-                    </span>
-                ) : (
-                    <></>
-                )}
-            </div>
-
             {version ? (
-                <div
-                    className={`font-semibold text-xs ${
-                        selected ? "text-gray-100 dark:text-gray-600" : "text-gray-600 dark:text-gray-500"
-                    } uppercase px-3 self-center`}
+                <span
```

Comment:
**suggestion**: This could be out of the scope here and could require some configuration with Tailwind (see [relevant docs](https://tailwindcss.com/docs/font-family)), but what do you think of using a `monospace` font for the version? Ideally, I'd propose going with the [JetBrains Mono](https://www.google.com/search?hl=en&q=jetbrains%20mono) font, but feel free to leave this out of the scope here and open a follow-up issue to keep track of this. 🏓

Reply:
I think it makes sense to use a monospaced font in this context, so let's do, why not 😄. Would it be ok with you to just use any of the built-in monospaced fonts (which we can simply leverage with `font-mono`) so that we don't have to worry about external fonts? 

From what I could find, the font itself in WOFF2 is 70kB, which is quite big for such a minor part of our UI.

@gtsiolis any strong reasons why the JetBrains Mono font in particular? If it's because of preference, I wanted to add that I really like https://github.com/tonsky/FiraCode and https://github.com/microsoft/cascadia-code as well!

Reply:
@filiptronicek:

> Would it be ok with you to just use any of the built-in monospaced fonts

It's generally preferred to rely on built-in system fonts for performance and more, see [relevant diff](https://github.com/gitpod-io/gitpod/commit/d7c7b928e3911c69b0011a04f559b2fe044b85fb#diff-73b930e56fa0314bf39f7b3c87fefbbb1b44fbdcd956fb2e775c14e628df557cR35-R51).

However, it'd be nice to try out web fonts to make the application render similarly across browsers and operating systems. Going with two font variants could suffice, and we can override font-weight on top of them. So far, we've been defaulting to `Inter` as the default font stack. I'd like to introduce `JetBrains Mono` for code-related elements.

> ... the font itself in WOFF2 is 70kB

Correct! Sounds ok. [FWIW](https://en.wiktionary.org/wiki/FWIW), other options like [Fira Code](https://github.com/tonsky/FiraCode) come with at least 130 KB.

> ... which is quite big for such a minor part of our UI.

I didn't mention that we need monospaced fonts generally in the dashboard UI for code-related elements, not just this part of the dashboard. 😅 

For example, **branch names** and **commit hashes** we surface on the dashboard could benefit from a monospaced font. However, feel free to leave this as is for now and open a follow-up issue to keep track of this. I'm OOO until mid-next week, but let's not block this from merging this. 🏓

Reply:
Let's use mono for now and force the JB Mono font later. Created https://github.com/gitpod-io/gitpod/issues/15443 for tracking 🙂